web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -53,16 +56,21 @@ export default {
       return !(this.name && this.description);
     },
   },
+  mounted() {
+    if (!loggedIn()) {
+      this.$router.push({name: 'home'});
+    }
+  },
   methods: {
     async register_dandiset() {
       const { name, description } = this;
```

Comment:
I realize you didn't write this, but this line is a little too clever for my liking. If you want to change it, feel free. If you disagree though, I don't feel strongly.

Reply:
I actually like this style, depending on the context, as if you're using the specified variables enough, it avoids the clutter of repeatedly specifying `this`. Whether it should specifically used here could be up for debate I suppose, but just wanted to give my input on that pattern in general.

Reply:
@AlmightyYakob sounds good.

Reply:
I personally prefer this pattern here. I tried removing it, and got this:
```
      const { data } = await girderRest.post('dandi', null, {
        params: {
          name: this.name,
          description: this.description,
        },
      });
```
To me the first pattern is more legible.